src/TodosContext.tsx

```diff
@@ -0,0 +1,209 @@
+/* eslint-disable no-console */
+import React, { useEffect, useState } from 'react';
+import { Filter } from './services/enums';
+import {
+  addTodo,
+  deleteTodo,
+  getTodos,
+} from './api/todos';
+import { Todo } from './types';
+
+const USER_ID = 11138;
+
+interface FilterParams {
+  filterBy?: Filter,
+}
+
+function filterTodosByCompleted(todos: Todo[], filter: Filter): Todo[] {
```

Comment:
can be moved to `utils` folder

Reply:
Not fixed
![image](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/54936044/414a5c39-e028-4f6d-8a70-2607674ff203)